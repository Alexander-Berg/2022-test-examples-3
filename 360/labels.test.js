'use strict';

const labelsMock = require('../../../test/mock/labels.json');
const labelsFilter = require('./labels.js');

describe('filters meta/labels', () => {
    it('работает', () => {
        expect(labelsFilter(labelsMock)).toMatchSnapshot();
    });
});
