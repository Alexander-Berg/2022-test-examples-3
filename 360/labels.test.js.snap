// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`работает 1`] = `
Array [
  Label {
    "color": "31c73b",
    "count": 51,
    "default": true,
    "lid": "11",
    "name": "4",
    "social": undefined,
    "symbolicName": "",
    "unused": undefined,
    "user": false,
  },
  Label {
    "color": "31c73b",
    "count": 33,
    "default": true,
    "lid": "12",
    "name": "54",
    "social": undefined,
    "symbolicName": "",
    "unused": undefined,
    "user": false,
  },
  Label {
    "color": "31c73b",
    "count": 26,
    "default": true,
    "lid": "2",
    "name": "answered",
    "social": undefined,
    "symbolicName": "answered_label",
    "unused": undefined,
    "user": false,
  },
  Label {
    "color": "31c73b",
    "count": 18,
    "default": true,
    "lid": "17",
    "name": "29",
    "social": undefined,
    "symbolicName": "",
    "unused": undefined,
    "user": false,
  },
  Label {
    "color": "31c73b",
    "count": 18,
    "default": true,
    "lid": "18",
    "name": "52",
    "social": undefined,
    "symbolicName": "",
    "unused": undefined,
    "user": false,
  },
  Label {
    "color": "31c73b",
    "count": 1,
    "default": true,
    "lid": "2140000004534368476",
    "name": "yamoney",
    "social": true,
    "symbolicName": "",
    "unused": undefined,
    "user": false,
  },
  Label {
    "color": "31c73b",
    "count": 0,
    "default": true,
    "lid": "1",
    "name": "12",
    "social": undefined,
    "symbolicName": "",
    "unused": undefined,
    "user": false,
  },
  Label {
    "color": "31c73b",
    "count": 0,
    "default": true,
    "lid": "3",
    "name": "draft",
    "social": undefined,
    "symbolicName": "draft_label",
    "unused": undefined,
    "user": false,
  },
  Label {
    "color": "31c73b",
    "count": 0,
    "default": true,
    "lid": "4",
    "name": "forwarded",
    "social": undefined,
    "symbolicName": "forwarded_label",
    "unused": undefined,
    "user": false,
  },
  Label {
    "color": "31c73b",
    "count": 0,
    "default": true,
    "lid": "5",
    "name": "mute",
    "social": undefined,
    "symbolicName": "mute_label",
    "unused": undefined,
    "user": false,
  },
  Label {
    "color": "31c73b",
    "count": 0,
    "default": true,
    "lid": "6",
    "name": "pinned",
    "social": undefined,
    "symbolicName": "pinned_label",
    "unused": undefined,
    "user": false,
  },
  Label {
    "color": "31c73b",
    "count": 0,
    "default": true,
    "lid": "7",
    "name": "priority_high",
    "social": undefined,
    "symbolicName": "important_label",
    "unused": undefined,
    "user": false,
  },
  Label {
    "color": "31c73b",
    "count": 0,
    "default": true,
    "lid": "8",
    "name": "remindme_threadabout:mark",
    "social": undefined,
    "symbolicName": "remindNoAnswer_label",
    "unused": undefined,
    "user": false,
  },
  Label {
    "color": "31c73b",
    "count": 0,
    "default": true,
    "lid": "9",
    "name": "encrypted",
    "social": undefined,
    "symbolicName": "encrypted_label",
    "unused": undefined,
    "user": false,
  },
  Label {
    "color": "31c73b",
    "count": 0,
    "default": true,
    "lid": "10",
    "name": "delayed_message",
    "social": undefined,
    "symbolicName": "delayed_message",
    "unused": undefined,
    "user": false,
  },
  Label {
    "color": "0",
    "count": 0,
    "default": false,
    "lid": "13",
    "name": "12312311",
    "social": undefined,
    "symbolicName": "",
    "unused": true,
    "user": true,
  },
  Label {
    "color": "99ffcc",
    "count": 0,
    "default": false,
    "lid": "14",
    "name": "afffff",
    "social": undefined,
    "symbolicName": "",
    "unused": true,
    "user": true,
  },
  Label {
    "color": "f79000",
    "count": 0,
    "default": false,
    "lid": "15",
    "name": "123121",
    "social": undefined,
    "symbolicName": "",
    "unused": true,
    "user": true,
  },
  Label {
    "color": "f79000",
    "count": 0,
    "default": false,
    "lid": "16",
    "name": "1231",
    "social": undefined,
    "symbolicName": "",
    "unused": true,
    "user": true,
  },
  Label {
    "color": "31c73b",
    "count": 0,
    "default": true,
    "lid": "2140000000006740116",
    "name": "odnoklassniki",
    "social": true,
    "symbolicName": "",
    "unused": undefined,
    "user": false,
  },
  Label {
    "color": "31c73b",
    "count": 0,
    "default": true,
    "lid": "FAKE_APPEND_LBL",
    "name": "FAKE_APPEND_LBL",
    "social": undefined,
    "symbolicName": "append_label",
    "unused": undefined,
    "user": false,
  },
  Label {
    "color": "31c73b",
    "count": 0,
    "default": true,
    "lid": "FAKE_ATTACHED_LBL",
    "name": "FAKE_ATTACHED_LBL",
    "social": undefined,
    "symbolicName": "attached_label",
    "unused": undefined,
    "user": false,
  },
  Label {
    "color": "31c73b",
    "count": 0,
    "default": true,
    "lid": "FAKE_COPY_LBL",
    "name": "FAKE_COPY_LBL",
    "social": undefined,
    "symbolicName": "copy_label",
    "unused": undefined,
    "user": false,
  },
  Label {
    "color": "31c73b",
    "count": 0,
    "default": true,
    "lid": "FAKE_DELETED_LBL",
    "name": "FAKE_DELETED_LBL",
    "social": undefined,
    "symbolicName": "deleted_label",
    "unused": undefined,
    "user": false,
  },
  Label {
    "color": "31c73b",
    "count": 0,
    "default": true,
    "lid": "FAKE_MULCA_SHARED_LBL",
    "name": "FAKE_MULCA_SHARED_LBL",
    "social": undefined,
    "symbolicName": "mulcaShared_label",
    "unused": undefined,
    "user": false,
  },
  Label {
    "color": "31c73b",
    "count": 0,
    "default": true,
    "lid": "FAKE_POSTMASTER_LBL",
    "name": "FAKE_POSTMASTER_LBL",
    "social": undefined,
    "symbolicName": "postmaster_label",
    "unused": undefined,
    "user": false,
  },
  Label {
    "color": "31c73b",
    "count": 0,
    "default": true,
    "lid": "FAKE_RECENT_LBL",
    "name": "FAKE_RECENT_LBL",
    "social": undefined,
    "symbolicName": "recent_label",
    "unused": undefined,
    "user": false,
  },
  Label {
    "color": "31c73b",
    "count": 0,
    "default": true,
    "lid": "FAKE_SEEN_LBL",
    "name": "FAKE_SEEN_LBL",
    "social": undefined,
    "symbolicName": "seen_label",
    "unused": undefined,
    "user": false,
  },
  Label {
    "color": "31c73b",
    "count": 0,
    "default": true,
    "lid": "FAKE_SPAM_LBL",
    "name": "FAKE_SPAM_LBL",
    "social": undefined,
    "symbolicName": "spam_label",
    "unused": undefined,
    "user": false,
  },
  Label {
    "color": "31c73b",
    "count": 0,
    "default": true,
    "lid": "FAKE_SYNCED_LBL",
    "name": "FAKE_SYNCED_LBL",
    "social": undefined,
    "symbolicName": "synced_label",
    "unused": undefined,
    "user": false,
  },
]
`;
